src/lights/LightShadow.js

```diff
@@ -107,15 +107,15 @@ Object.assign( LightShadow.prototype, {
 
 		return this;
 
-	},
+	}
 
-	clone: function () {
+	clone() {
 
-		return new this.constructor().copy( this );
+		return new LightShadow().copy( this );
```

Comment:
After long time of debugging, I've realized that this line is responsible for the unit test breakage. 

Can you please explain why have you changed it?

Reply:
...they're the same thing? what do you mean? one is more verbose than the other.

Reply:
They are not the same thing. I suggest you only modify the code which is absolutely necessary for the ES6 migration.

Reply:
Well that's weird to me.